pdr_backend/util/time_types.py

```diff
@@ -132,7 +133,42 @@ def pretty_timestr(self) -> str:
 
 
 @enforce_types
-def _dt_now_UTC() -> datetime:
-    dt = datetime.utcnow()
-    dt = dt.replace(tzinfo=timezone.utc)  # tack on timezone
-    return dt
+def timestr(dt: datetime) -> str:
+    """Simple time string, useful for testing"""
+    return dt.strftime("%Y-%m-%d %H:%M:%S")
+
+
+@enforce_types
+def tz_offset_from_utc(delta_hours: int):
```

Comment:
as far as I can tell, this function only serves the purpose of defining a constant for the tests (`TZ_UTC_MINUS_5`). I think it would be better if we removed this function. In the tests, we can simply define the tested timezone as ""America/New_York" or something similar.

Alternately, we could at least move this function in some test resources/utils part for it to be clearly delimited from main code.

Reply:
That makes sense. This PR does that: https://github.com/oceanprotocol/pdr-backend/pull/1344